Code/GraphMol/DistGeomHelpers/BoundsMatrixBuilder.cpp

```diff
@@ -1156,11 +1398,15 @@ void set14Bounds(const ROMol &mol, DistGeom::BoundsMatPtr mmat,
       donePaths[id1] = 1;
       donePaths[id2] = 1;
 
-      if (rSize > 5) {
+      if (rSize > 5 && rSize < 9) {
```

Comment:
Just to make this more explicit and easier to change later if we need to, I would suggest defining a `static const double minMacrocycleRingSize=9` in the global scope of this file and then use that everywhere you're currently using `9` in this file

Reply:
done